conf.yaml

```diff
@@ -489,14 +489,15 @@ contents:
         title:      Kibana: Explore, Visualize, and Share
         sections:
           -
-            title:      Kibana Reference
```

Comment:
I'm not adjusting the indentation of the other lines because I think we'll remove all of the `asciidoctor: true` lines when we convert all of the docs.